roles/ceph-restapi/tasks/docker/dirs_permissions.yml

```diff
@@ -9,12 +9,7 @@
 - set_fact:
     after_hamer=True
   when:
-    ceph_version not in (firefly or giant or hammer)
-
-- set_fact:
-    after_hamer=False
-  when:
-    ceph_version in (firefly or giant or hammer)
```

Comment:
I copied what was done in dirs_permisssions.yml for ceph-mon and ceph-osd.  I assumed that that was the way to do it correctly.